apbs/chemistry/atom_list.py

```diff
@@ -52,8 +56,7 @@ def mincrd(self) -> Coordinate:
         return self._dp["min"]
 
     def maxcrd(self) -> Coordinate:
-        """Maximum coordinates
-        """
+        """Maximum coordinates"""
```

Comment:
FWIW, I would have named this function (and its "mincrd" equivalent) something like `max_coord` to make the code a bit more readable.